foam_internalwrapper/cuba_extension.py

```diff
@@ -0,0 +1,20 @@
+""" Provisional CUBA keywords specific for this revision
```

Comment:
this file ` foam_internalwrapper/cuba_extension.py` is identical to `foam_controlwrapper/cuba_extension.py`.  There should be a **single** module that **both** the FILE-IO and INTERNAL wrapper uses.